docs/conf.py

```diff
@@ -14,6 +14,9 @@
 import sys
 
 sys.path.insert(0, os.path.abspath("../opentelemetry-api/src/"))
+sys.path.insert(
```

Comment:
I think it would look nicer to merge these two statements into one like `sys.path[:0] = [os.path.abspath("../opentelemetry-api/src/"), os.path.abspath("../ext/opentelemetry-ext-opentracing-shim/src/")]`.

Reply:
Thanks! Done.